tests/load-tests.php

```diff
@@ -237,23 +237,130 @@ function( $module_settings, $module_dir ) {
 		);
 	}
 
-	public function test_perflab_maybe_set_object_cache_dropin() {
-		if ( ! $GLOBALS['wp_filesystem'] && ! WP_Filesystem() ) {
-			$this->markTestSkipped( 'Filesystem cannot be initialized.' );
-		}
+	public function test_perflab_maybe_set_object_cache_dropin_no_conflict() {
+		global $wp_filesystem;
 
-		if ( ! $GLOBALS['wp_filesystem']->is_writable( WP_CONTENT_DIR ) ) {
-			$this->markTestSkipped( 'This system does not allow file modifications within WP_CONTENT_DIR.' );
-		}
+		$this->set_up_mock_filesystem();
+
+		// Ensure PL object-cache.php drop-in is not present and constant is not set.
+		$this->assertFalse( $wp_filesystem->exists( WP_CONTENT_DIR . '/object-cache.php' ) );
+		$this->assertFalse( PERFLAB_OBJECT_CACHE_DROPIN_VERSION );
```

Comment:
Got an error when run unit tests in local system.

`Failed asserting that 1 is false.`

Reply:
@mukeshpanchal27 Is that maybe because your test setup still had the actual `object-cache.php` file present from some older code? Can you try checking that in your test setup, and if so delete the file and rerun?

Reply:
@mukeshpanchal27 testing this locally myself I did not come across this error, all tests passed.

Reply:
Thanks for checking. Working now.